pkg/karmadactl/utils/template.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+)
+
+// ParseTemplate validates and parses passed as argument template
+func ParseTemplate(strTmpl string, obj interface{}) ([]byte, error) {
```

Comment:
Whether this feature is versatile enough？Should we put it to `pkg/karmadactl/addons/utils`?

Reply:
> Whether this feature is versatile enough？Should we put it to `pkg/karmadactl/addons/utils`?

Yes，I am not certain it will be reused future and put it here at first and waiting your suggestions.
I will move it.

Reply:
Put it `pkg/karmadactl/addons/utils` firstly, if it need to be used by other modules later, we will consider  that move it to shared pkg.